config/routes.rb

```diff
@@ -1,7 +1,10 @@
 Rails.application.routes.draw do
+  get 'users/show'
   devise_for :users
 
   resources :dishes
   resources :menu_days, only: %i[index show]
+  resources :orders, only: %i[index new create]
```

Comment:
You don't add show here, but you added show method in controller! Please change it.
1. add this method usage
2. remove method, view, route